draft-ietf-quic-transport.md

```diff
@@ -2097,6 +2100,9 @@ change in the initial limits (see {{zerortt-parameters}}).
 The MAX_STREAM_DATA frame (type=0x05) is used in flow control to inform a peer
 of the maximum amount of data that can be sent on a stream.
 
+An endpoint that receives a MAX_STREAM_DATA frame for a receive-only stream
+MUST terminate the connection with error PROTOCOL_VIOLATION.
+
```

Comment:
What about when receiving MAX_STREAM_DATA on a send or bidi stream but that has not yet been opened?

Reply:
MAX_STREAM_DATA is defined in the state machine as one way to open a stream.

Reply:
In section 10.2.3 it says "The receiver only sends MAX_STREAM_DATA in the “Recv” state. " So I assume it would be a protocol error to send it in other states.

Reply:
Ah, I see now that this is for the receive end of bidirectional streams only. So, yes, you're right.